includes/class-sensei-admin.php

```diff
@@ -534,6 +534,12 @@ private function duplicate_content( $post_type = 'lesson', $with_lessons = false
 			wp_die( esc_html__( 'Invalid post type. Can duplicate only lessons and courses', 'sensei-lms' ) );
 		}
 
+		// Set up event properties.
+		$event            = $post_type . '_duplicate';
```

Comment:
This is tempting, but one of the guidelines for event naming conventions is to not build them up programmatically. It is in this doc page - PCYsg-4S2-p2

> Don’t programmatically create event names. They should be hardcoded so they are easily searchable in the code base.

So it would be better if we could make the event name explicit. Note that we do dynamically add the `sensei_` prefix as an exception.

Reply:
9d71cbde03b2bcdf420b007d68c33e3cb4528fb5